win32ss/user/winsrv/usersrv/init.c

```diff
@@ -140,7 +140,7 @@ CSR_API(SrvGetThreadConsoleDesktop)
 {
     PUSER_GET_THREAD_CONSOLE_DESKTOP GetThreadConsoleDesktopRequest = &((PUSER_API_MESSAGE)ApiMessage)->Data.GetThreadConsoleDesktopRequest;
 
-    DPRINT1("%s not yet implemented\n", __FUNCTION__);
+    DPRINT("%s not yet implemented\n", __FUNCTION__);
```

Comment:
Does this display so many times?

Reply:
not as often as the floating point stuff, but still enough to cause severe lag.